ospd/misc.py

```diff
@@ -216,6 +216,8 @@ def create_scan(self, scan_id='', targets='', options=None, vts=''):
             and self.id_exists(scan_id)
             and (self.get_status(scan_id) == ScanStatus.STOPPED)
         ):
+            self.scans_table[scan_id]['end_time'] = "0"
```

Comment:
Shouldn't it be 
```suggestion
            self.scans_table[scan_id]['end_time'] = 0
```
?